### ISSUE TITLE
unable to checkpoint weights after certain loss improvement

### ISSUE THREAD

**Issue by syedalirazataqvi** [keras-team/keras#14305]
Hi all, I am saving my model check point in hdf5 format using following code:

 ` regressor.load_weights(CHEKCHPOINT_FILE_PATH)
 
  opt = keras.optimizers.adam(learning_rate=0.0001)
  regressor.compile(optimizer = opt, loss = 'mean_squared_error', metrics=['accuracy'])

  checkpoint = ModelCheckpoint(CHEKCHPOINT_FILE_PATH, monitor='loss', verbose=1, save_best_only=True, mode='min')
  callbacks_list = [checkpoint]
  regressor.fit(train_images, train_labels, epochs = 2000, batch_size = 100, callbacks=callbacks_list, verbose=0)
`
 

initially this code was saving weights, but after the loss of 0.00003 is stopped saving and always starts from the loss of 0.00004

in the following log loss has decreased to 0.00003.

Epoch 00001: loss improved from inf to 0.00004, saving model to ./drive/MyDrive/ColabNotebooks/FXModel.weights.best.hdf5

Epoch 00002: loss improved from 0.00004 to 0.00004, saving model to ./drive/MyDrive/ColabNotebooks/FXModel.weights.best.hdf5

Epoch 00003: loss improved from 0.00004 to 0.00004, saving model to ./drive/MyDrive/ColabNotebooks/FXModel.weights.best.hdf5

Epoch 00004: loss improved from 0.00004 to 0.00004, saving model to ./drive/MyDrive/ColabNotebooks/FXModel.weights.best.hdf5

Epoch 00005: loss improved from 0.00004 to 0.00003, saving model to ./drive/MyDrive/ColabNotebooks/FXModel.weights.best.hdf5

Epoch 00006: loss did not improve from 0.00003

Epoch 00007: loss did not improve from 0.00003

Epoch 00008: loss did not improve from 0.00003

Epoch 00009: loss did not improve from 0.00003

Epoch 00010: loss did not improve from 0.00003

Epoch 00011: loss did not improve from 0.00003

Epoch 00012: loss did not improve from 0.00003

Epoch 00013: loss improved from 0.00003 to 0.00003, saving model to ./drive/MyDrive/ColabNotebooks/FXModel.weights.best.hdf5

but next time hen i started training it started from 0.00004:

Epoch 00001: loss improved from inf to 0.00004, saving model to ./drive/MyDrive/ColabNotebooks/FXModel.weights.best.hdf5

please help.

**Comment by sushreebarsa** [comment]
@syedalirazataqvi Moving this issue to closed status as there has been no recent activity.In case you still face the error please create a new issue,we will get you the right help.Thanks!
